### PR TITLE
Add rusk-abi as member of the workspace

### PR DESCRIPTION
**Pull request by ZER0**

### CODE REVIEW

File: rusk-abi/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+- Add ABI infrastracture
+- Add Poseidon Hash host function
+- Add test contract
+- Add CHANGELOG.md
+- Add LICENSE
+- Add README.md
+
+[unreleased]: https://github.com/dusk-network/dusk-abi/compare/v0.1.0...HEAD
+[0.1.0]: https://github.com/dusk-network/dusk-abi/releases/tag/v0.1.0
```
Comment (CPerezz) [inline rusk-abi/CHANGELOG.md:22]: Not sure what;s this but is not inline with  the rest of our CHANGELOG formats.
Also, add newline please.

**Review by CPerezz — APPROVED** [review]
The general workspace makefile should call this one in the `test` exec

File: rusk-abi/CHANGELOG.md (hunk 1 above)
Comment (ZER0) [reply rusk-abi/CHANGELOG.md:22]: It's our CHANGELOG format that is not inline with this. The other CHANGELOG claimed to use the "Keep a Changelog" but it's not. See: https://keepachangelog.com/en/1.0.0/

For example the section "Unreleased" is totally useless in that way. All the CHANGELOG I made are following the proper format, and makes all the version linked properly to the actual changes, released or not.

For example many CHANGELOG didn't use the proper date format and I already updated that to ISO 8601. Soon or later we should fix the other CHANGELOG, but it's a work that requires time. So at least for the new ones we're doing in the right way from the start.

Comment (ZER0) [reply rusk-abi/CHANGELOG.md:22]: BTW about the new line, the CHANGELOG has one, so I'm not sure what you're referring to.

**Review by CPerezz — APPROVED** [review]
LGTM

**Pull request merged**